README.md

```diff
@@ -224,6 +224,15 @@ The fetch interval in seconds between retrievals of signed API data.
 
 A list of signed API URLs to call along with URLs fetched from the chain.
 
+#### `walletDerivationScheme`
+
+The derivation scheme used to derive sponsor wallets. The following options are available:
```

Comment:
Doesn't feel future compatible with https://api3workspace.slack.com/archives/C05TQPT7PNJ/p1706712947591099?thread_ts=1706521432.528779&cid=C05TQPT7PNJ

Reply:
Yeah, but does it really need to be? I don't think it's a big issue if the Airseeker config changes and I wanted to prioritise clarity for current implementation. That said, if you want it future proof, we might do and object right away:
- `{ type: "managed" }`
- `{ type: "self-funded" }`
- (In the future) `{ type: "fixed-wallet", "mnemonic": "..." }` (mnemonic may not be needed if we use the Airseeker one in which case we could get away with just union of strings).

Reply:
Yeah, add the `{}`, it's definitely worth it